e2e/scenario_nvidia_device_plugin_test.go

```diff
@@ -0,0 +1,116 @@
+package e2e
+
+import (
+	"context"
+	"testing"
+
+	"github.com/Azure/agentbaker/e2e/config"
+	"github.com/Azure/agentbaker/pkg/agent/datamodel"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/compute/armcompute/v6"
+)
+
+func Test_Ubuntu2404_NvidiaDevicePluginRunning(t *testing.T) {
```

Comment:
Can we also add a test for MIG enabled nodes? And we should test it for mixed strategy.

Reply:
Mixed strategy is not applicable at this time, as we are not supporting that.